R/render.R

```diff
@@ -1,15 +1,28 @@
-epi_from_subpopulation <- function(target, age, lower, upper) {
+count_of_subpopulation <- function(target, age, lower, upper) {
   in_range <- individual::Bitset$new(target$max_size)$insert(
     which((age >= lower) & (age <= upper))
   )
+  n_in_range <- in_range$size()
+  if (n_in_range == 0) {
+    return(0)
+  }
+  n_in_range
+}
```

Comment:
Do we need `target` anymore in this function?

Reply:
I think so? so we can calculate the max size of the group in which the detected people come from?